examples/scroll-sync.amp.html

```diff
@@ -22,7 +22,10 @@
   </div>
 
   <div>
-    <amp-fx-scroll-sync name='scale'>
+    <amp-fx-scroll-sync name='scale'
```

Comment:
My thinking might be wrong, but I'd think that for scaling, we need only `end-scale`, at which position to start scaling and the direction of scrolling. Right?